saleor/graphql/product/mutations/products.py

```diff
@@ -236,7 +236,7 @@ class CollectionReorderProducts(BaseMutation):
     )
 
     class Meta:
-        description = "Reorder the products of a collection"
+        description = "Reorder the products of a collection."
```

Comment:
To be consistent `collection` should start from the upper letter. See 👇

Reply:
Fixed, but I made them all lowercase.